Week1_Knowledge.md

```diff
@@ -77,6 +77,8 @@ Conclusion: If _P_ is false, then _Q_ can be anything.
 Equivalent to:
 _P_ → _Q_ ∧ _Q_ → _P_
 
+Only true if P and Q are the same.
```

Comment:
Maybe change this to say, "Biconditionals evaluate to true only if P and Q are the same," because right now it seems like you're saying the expression: _P_ → _Q_ ∧ _Q_ → _P_ itself is only valid if P and Q are true.